src/pages/en/blog/2023-11-30-quilt-update.md

```diff
@@ -0,0 +1,38 @@
+
+## Mappings
+
+As always, QM has been updated to each and every snapshot this month, which includes `23w44a`, `23w45a`, `23w46a`, `1.20.3-pre1`, and `1.20.3-pre2`. Though new mappings have been a tad scarce, we saw some [important entity names](https://github.com/QuiltMC/quilt-mappings/pull/513
+) from [HyperPidgeon](https://github.com/HyperPigeon), and a few [changes to item mappings](https://github.com/QuiltMC/quilt-mappings/pull/514) by yours truly to account for the changes Mojang made to accomodate the [Crafter](https://minecraft.wiki/w/Crafter). Along with those two small pull requests, I brought you [full coverage of `net.minecraft.data`](https://github.com/QuiltMC/quilt-mappings/pull/515) and [Platymemo](https://github.com/Platymemo) delivered [a huge update to advancement mappings](https://github.com/QuiltMC/quilt-mappings/pull/496).
+
+Since the main team is busy toiling away on [Enigma](https://github.com/QuiltMC/enigma/)'s `2.0` update, QM has lost coverage this month, going from `94.42%` in the final `23w44a` version to `93.15%` in the current build of `1.20.3-pre2`.  Quilt Mappings development is incredibly simple to pick up, and anyone with experience developing Minecraft mods already has the knowledge needed to start mapping! You can read our [blog post](https://quiltmc.org/en/blog/2023-06-03-qm-needs-you/) for more info about contributing to QM or go straight to our [contributing guide](https://github.com/QuiltMC/quilt-mappings/blob/HEAD/CONTRIBUTING.md) to help!
+
+Now: what is that `2.0` update that the team is so caught up with? Enigma is our *deobfuscator*, the program that we use to create mappings for Minecraft. It allows you to see and navigate Minecraft's obfuscated code, as well as apply names you come up with. This month we've been working on the final improvements to its plugin system, shipping two big changes. First, Iota added new parsing features for profiles, with allow you to pass in data to plugins. In QM we use this to pass in information on Minecraft classes to [our plugin](https://github.com/QuiltMC/quilt-enigma-plugin). Iota's [pull request](https://github.com/QuiltMC/enigma/pull/158) allows you to use a JSON list instead of an unwieldy string object, making parsing simpler for plugins and allowing JSON validation to act on lists.
+
+Secondly, I worked on a [huge pull request](https://github.com/QuiltMC/enigma/pull/163) which overhauled the *name proposal API*, a feature which allows plugins to automatically name entries. We redesigned the way that plugins propose names, added tons of documentation, and added a priority system: the order that you declare proposal services in your profile will now dictate which plugins take priority in case two plugins propose a name for the same entry. Additionally, it fixed some crashes, fixed some bugs, removed some Recaf mapping format support, updated some other APIs, and cleaned up and documented internals as always.
```

Comment:
```suggestion
Secondly, I worked on a [huge pull request](https://github.com/QuiltMC/enigma/pull/163) which overhauled the *name proposal API*, a feature which allows plugins to automatically name entries. This means that an Enigma plugin is able to analyse the bytecode, or compiled Java code, of the project's files and guess, or _propose_ as we call it, names for things that it understands the purpose of based on the code. This is incredibly useful for repetitive code, and reduces the workload for the humans who work on QM. We redesigned the way that plugins propose names, added tons of documentation, and added a priority system: the order that you declare proposal services, the sections of your plugin that make automatic names, in your profile will now dictate which plugins take priority in case two plugins propose a name for the same entry. Additionally, it fixed some crashes, fixed some bugs, removed some Recaf mapping format support, updated some other APIs, and cleaned up and documented internals as always.
```

Reply:
It wouldn't let me use the suggestion editor for some reason